src/contracts/protocol/pool/Pool.sol

```diff
@@ -450,6 +450,10 @@ abstract contract Pool is VersionedInitializable, PoolStorage, IPool {
     res.accruedToTreasury = reserve.accruedToTreasury;
     res.unbacked = reserve.unbacked;
     res.isolationModeTotalDebt = reserve.isolationModeTotalDebt;
+    // This is a temporary workaround for integrations that are broken by Aave 3.2
+    // While the new pool data provider is backward compatible, some integrations hard-code an old implementation
+    // To allow them to unlock the funds, the pool address provider is setting a stable debt token, so balanceOf() and totalSupply() will return zero instead of reverting
```

Comment:
```suggestion
    // To allow them to not have any infrastructural blocker, a mock must be configured in the Aave Pool Addresses Provider, returning zero on all required view methods, instead of reverting
```